File: library/src/main/java/com/a65apps/kalimullinilnazrafilovich/library/applicaiton/db/AppDatabase.java
package com.a65apps.kalimullinilnazrafilovich.library.applicaiton.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.a65apps.kalimullinilnazrafilovich.library.applicaiton.interfaces.LocationDao;
import com.a65apps.kalimullinilnazrafilovich.library.applicaiton.models.LocationOrm;

import io.reactivex.rxjava3.annotations.NonNull;

@Database(entities = {LocationOrm.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
    @NonNull
    public abstract LocationDao locationDao();
}

File: library/src/main/java/com/a65apps/kalimullinilnazrafilovich/library/applicaiton/db/AppDatabase.kt
package com.a65apps.kalimullinilnazrafilovich.library.applicaiton.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.a65apps.kalimullinilnazrafilovich.library.applicaiton.interfaces.LocationDao
import com.a65apps.kalimullinilnazrafilovich.library.applicaiton.models.LocationOrm

@Database(entities = [LocationOrm::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun locationDao(): LocationDao
}
